data-prepper-plugins/http-source/src/test/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceTest.java

```diff
@@ -186,6 +188,16 @@ public void cleanUp() {
         }
     }
 
+    private void assertSecureResponseWithStatusCode(final AggregatedHttpResponse response, final HttpStatus status) {
+        Assertions.assertEquals(status, response.status());
```

Comment:
We are generally using Hamcrest matchers going forward. But, this is fine for now.